src/lazy/raw_stream_item.rs

```diff
@@ -109,28 +109,35 @@ impl<M: Copy + Debug, V: Copy + Debug, E: Copy + Debug> RawStreamItem<M, V, E> {
     }
 }
 
+/// Represents the end of a [`RawInputStream`].
```

Comment:
FYI—there's a GHA annotation on this line of this comment that says that `RawInputStream` is an unresolved link.